src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/User/Edit.html

```diff
@@ -45,9 +45,8 @@
         <input
             type="password"
             name="password_repeat"
-            data-ng-model="data.passwordRepeat"
-            data-ng-class="{'ng-invalid': (input.password !== input.passwordRepeat) && userEditForm.password_repeat.$dirty}"/>
-        <span class="input-error" data-ng-show="(input.password !== input.passwordRepeat) && userEditForm.password_repeat.$dirty">
+            data-ng-model="data.passwordRepeat" />
```

Comment:
No, in this one.  See https://github.com/liqd/adhocracy3/pull/2687/commits/01ca41e54b403efc307e1a84e6edf35d6c0b55e7#diff-38f88590d3a2d5e444e2bd4710f305ffR776